### DISCUSSION TITLE
Example of programmatic verification email

### DISCUSSION THREAD

**Discussion by seanbjornsson** [jeremyevans/rodauth#315] [General]
In Version 2.6.0 `setup_account_verification` was promoted to public api with the note "You can use this method for automatically sending account verification emails when automatically creating accounts."

Does anyone have any example code for this action? I thought this might be used via the Internal Request approach, but it's not listed in the documentation [there](https://github.com/jeremyevans/rodauth/blob/master/doc/internal_request.rdoc).

**Comment by jeremyevans** [comment]
This stemmed from a request on the Google Group: https://groups.google.com/g/rodauth/c/CrRlaOgGAuA/m/0t7Ky2b6AgAJ

You are correct that this isn't currently exposed by the internal request feature.  I suppose it could be if you want to add support and tests for it.

**Comment by seanbjornsson** [comment]
Ah yes thank you for pointing that out.
The part I was missing was calling `account_from_login` first.

And just to leave this here for posterity, another example outside of that google forum discussion:
https://github.com/enterprise-oss/osso-rb/blob/main/lib/osso/graphql/mutations/invite_admin_user.rb#L43
